input/blog/posts/_current-newsletter-draft.md

```diff
@@ -17,15 +17,23 @@ Welcome to the .NET Foundation update for March & April. Every other month, we'l
 
 ### [Project Spotlight](https://dotnetfoundation.org/projects/spotlight)
 
+### [.NEXT](https://github.com/sakno/dotNext) 
```

Comment:
Can you or @ChrisSfanos get the logos for these and put them in the table format like the other new projects?